server/models/Message.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const messageSchema = new mongoose.Schema(
+  {
+    sender: {
+      type: mongoose.Types.ObjectId,
+      ref: "Profile",
+      required: [true, "A message must contain the sender"],
+    },
+    conversationId: {
+      type: mongoose.Types.ObjectId,
+      ref: "Conversation",
+      required: [true, "A message must contain the referenced conversation"],
+    },
```

Comment:
As an aside, it would make sense to consider [indexing](https://en.wikipedia.org/wiki/Database_index#:~:text=Indexes%20are%20used%20to%20quickly,efficient%20access%20of%20ordered%20records.) this field since we would almost always query messages with their `conversationId`. This is fairly advanced and potentially premature, though it's worth being aware 👍🏼

Reply:
Hrmmm, I've never thought to do this in mongodb, but will look into it and implement indexing in this request.